packages/obonode/obojobo-sections-assessment/components/full-review/index.js

```diff
@@ -138,6 +142,13 @@ class AssessmentReviewView extends React.Component {
 			)
 		})
 
+		// Deciding which is the highest attempt score
+		attempts.forEach(attempt => {
```

Comment:
I think this may not be necessary, since you have https://github.com/ucfopen/Obojobo/pull/1617/files#diff-91c983cfa647c49b998f8eb80445c3adbf590c93323345f792c6429e02d38bf3R30 - That line should give you an array of highest attempts, and may in the end be the only change needed in the PR.

Reply:
That's true. I could have handled it just using _highestAttempts_. Working on it!